input/community/speakers/michael-dera.md

```diff
@@ -0,0 +1,29 @@
+---
+Title: Michael Dera
+Pronouns: He
```

Comment:
Build failure, Preferred gender pronouns should be of the form [subject]/[object] in lowercase (I.e. "they/them") (ValidateSpeakerPronouns)